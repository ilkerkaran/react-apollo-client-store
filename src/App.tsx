import React from 'react';
import './App.css';
import FormSample from './containers/FormSample/FormSample';

function App() {
  return (
    <div className="App">
      <FormSample />

    </div>
  );
}

export default App;
