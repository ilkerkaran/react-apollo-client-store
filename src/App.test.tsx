import React from 'react';
import { create } from 'react-test-renderer';
import App from './App';

test('renders learn react link', () => {
  const a = create(<App />);
  expect(a).toMatchSnapshot();
});
