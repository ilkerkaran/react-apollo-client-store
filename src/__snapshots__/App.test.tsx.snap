// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  className="App"
>
  <div
    className="Form"
  >
    <h4>
      Form Title
    </h4>
    <form>
      <button
        className="Button Success"
        onClick={[Function]}
        type="button"
      >
         
        Send
      </button>
    </form>
  </div>
</div>
`;
